### DISCUSSION TITLE
2.5.4.0 release date?

### DISCUSSION THREAD

**Discussion by anton-kurilov** [ESAPI/esapi-java-legacy#834] [General]
Hey all, when is the 2.5.4.0 planned to be tagged? Would be nice to get https://github.com/ESAPI/esapi-java-legacy/pull/833 resolved when pulling from maven repo. Thanks!

**Comment by tstegen** [comment]
Can we have feedback on the deployment of the next update ?

**Reply by kwwall** [reply]
Waiting on requested changes on a PR. I've reached out to the contributor? Otherwise I will have to leave out PR #813.

**Reply by kwwall** [reply]
I will be releasing this today or tomorrow. I have decided to not wait for PR #813.

**Reply by kwwall** [reply]
Definitely won't be tonight. I'm getting a migraine and also ran into a Maven convergence issue which is hard to think through with this splitting headache. I'll give it another spin tomorrow after work.

**Reply by tstegen** [reply]
thank you for your feedback and take care of yourself
